doc/source/myrocks/variables.rst

```diff
@@ -2508,6 +2530,17 @@ are positive integers. The block accesses are saved to
 the ``rocksdb_datadir/block_cache_traces/trace_file_name``.
 The default value is ``""``, an empty string.
 
+.. variable:: rocksdb_trace_queries
+
+  :version 5.7.35-38: Implemented
+  :cli: ``--rocksdb-trace-queries``
+  :dyn: Yes
+  :scope: Global
+  :vartype: String
+  :default: " "
+
+Trace option string. The format is sampling_frequency:max_trace_file_size:trace_file_name. The sampling_frequency value and max_trace_file_size value are positive integers. The queries are saved to the rocksdb_datadir/queries_traces/trace_file_name.
```

Comment:
My fault, I should have included this information in the ticket.

It might be useful commenting the units:
- The unit of the file size is bytes
- The sampling frequency specifies that one request is sampled from sampling_frequency requests. If is 1, all the requests are traced. If is 5, it means 1 request will be trace of every 5.

Reply:
Added.